core/protobuf/src/main/java/alluxio/proto/ProtoMessage.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.proto;
+
+import alluxio.proto.dataserver.Protocol;
+
+import com.google.protobuf.InvalidProtocolBufferException;
+import com.google.protobuf.MessageLite;
+
+/**
+ * A simple wrapper around Protobuf MessageLite.
+ */
+public final class ProtoMessage {
```

Comment:
Javadoc the methods in this class? Also, is this intended to be used internally only (ie. other code bases should not try to import our shaded protobuf package)?